Panicked while decrypt passwords data

**Describe the bug**
A clear and concise description of what the bug is.
描述一下遇到的 Bug，和对应的报错信息 `./hack-browser-data -vv`

Unable to decrypt password, everything display: `parse.go:639: debug https://REDACTED/ have empty password decrypt failed, password is empty`.

Then its panic:
```log
...
...
...
// SNIPPED
parse.go:639: debug https://REDACTED/ have empty password decrypt failed, password is empty
parse.go:639: debug https://REDACTED/ have empty password decrypt failed, password is empty
parse.go:639: debug https://REDACTED/ have empty password decrypt failed, password is empty
...
panic: runtime error: slice bounds out of range [:-81]

goroutine 1 [running]:
hack-browser-data/core/decrypt.PKCS5UnPadding(...)
        /root/Tools/HackBrowserData/core/decrypt/decrypt.go:176
hack-browser-data/core/decrypt.aes128CBCDecrypt(0xc000238750, 0x10, 0x14, 0xc0004cd2d0, 0x10, 0x10, 0xc0009ca183, 0x20, 0x20, 0x41acb8, ...)
        /root/Tools/HackBrowserData/core/decrypt/decrypt.go:169 +0x1b1
hack-browser-data/core/decrypt.ChromePass(0xc000238750, 0x10, 0x14, 0xc0009ca180, 0x23, 0x23, 0xc0004cd220, 0xf, 0x10, 0x0, ...)
        /root/Tools/HackBrowserData/core/decrypt/decrypt_linux.go:9 +0x10a
hack-browser-data/core/data.(*passwords).ChromeParse(0xc000012e80, 0xc000238750, 0x10, 0x14, 0x0, 0x0)
        /root/Tools/HackBrowserData/core/data/parse.go:636 +0x70a
hack-browser-data/cmd.Execute.func1(0xc000012840, 0x0, 0xe)
        /root/Tools/HackBrowserData/cmd/cmd.go:86 +0x5c2
github.com/urfave/cli/v2.(*App).RunContext(0xc000001980, 0x885b30, 0xc0000181e0, 0xc000012080, 0x4, 0x4, 0x0, 0x0)
        /root/.go/pkg/mod/github.com/urfave/cli/v2@v2.2.0/app.go:315 +0x6fe
github.com/urfave/cli/v2.(*App).Run(...)
        /root/.go/pkg/mod/github.com/urfave/cli/v2@v2.2.0/app.go:215
hack-browser-data/cmd.Execute()
        /root/Tools/HackBrowserData/cmd/cmd.go:115 +0x7c5
main.main()
        /root/Tools/HackBrowserData/main.go:8 +0x25
```


**Desktop (please complete the following information):**
 - OS Name 操作系统名称: Ubuntu 16.04.7 LTS
 - Browser Name 浏览器名称: Chrome
 - Browser Version 浏览器版本: Version 91.0.4472.114 (Official Build) (64-bit)

**Additional context**
Add any other context about the problem here.
其他有用的信息

--- a/core/decrypt/decrypt.go
+++ b/core/decrypt/decrypt.go
@@ -19,6 +19,7 @@ var (
 	errSecurityKeyIsEmpty = errors.New("input [security find-generic-password -wa 'Chrome'] in terminal")
 	errDecryptFailed      = errors.New("decrypt failed, password is empty")
 	errDecodeASN1Failed   = errors.New("decode ASN1 data failed")
+	errEncryptedLength    = errors.New("length of encrypted password less than block size")
 )
 
 type ASN1PBE interface {
@@ -164,7 +165,7 @@ func aes128CBCDecrypt(key, iv, encryptPass []byte) ([]byte, error) {
 	}
 	encryptLen := len(encryptPass)
 	if encryptLen < block.BlockSize() {
-		return nil, err
+		return nil, errEncryptedLength
 	}
 
 	dst := make([]byte, encryptLen)

--- a/core/decrypt/decrypt.go
+++ b/core/decrypt/decrypt.go
@@ -17,7 +17,6 @@ import (
 
 var (
 	errSecurityKeyIsEmpty = errors.New("input [security find-generic-password -wa 'Chrome'] in terminal")
-	errPasswordIsEmpty    = errors.New("password is empty")
 	errDecryptFailed      = errors.New("decrypt failed, password is empty")
 	errDecodeASN1Failed   = errors.New("decode ASN1 data failed")
 )
@@ -163,7 +162,12 @@ func aes128CBCDecrypt(key, iv, encryptPass []byte) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
-	dst := make([]byte, len(encryptPass))
+	encryptLen := len(encryptPass)
+	if encryptLen < block.BlockSize() {
+		return nil, err
+	}
+
+	dst := make([]byte, encryptLen)
 	mode := cipher.NewCBCDecrypter(block, iv)
 	mode.CryptBlocks(dst, encryptPass)
 	dst = PKCS5UnPadding(dst)

